components/elevation/stories/elevation.stories.js

```diff
@@ -26,6 +26,7 @@ const styles = () => html`
 			padding: 20px;
 			text-align: center;
 			width: 500px;
+			border-radius: 6px;
```

Comment:
why not using our card?

Reply:
because card comes with elevation :)
You suggested that we will drop the elevation from the storybook - this will be done once this and https://github.com/Vonage/vivid/pull/1273 will be merged